docker-compose.prod.yml

```
@@ -0,0 +1,90 @@
+    environment:
+      - N8N_HOST=n8n.speedandfunction.com
+      - N8N_PROTOCOL=https
+      - N8N_SSL_KEY=/home/node/certs/privkey.pem
+      - N8N_SSL_CERT=/home/node/certs/fullchain.pem
+      - N8N_GOOGLE_CLIENT_ID=${GOOGLE_CLIENT_ID:?GOOGLE_CLIENT_ID is required}
+      - N8N_GOOGLE_CLIENT_SECRET=${GOOGLE_CLIENT_SECRET:?GOOGLE_CLIENT_SECRET is required}
+      - N8N_GOOGLE_CALLBACK_URL=${N8N_GOOGLE_CALLBACK_URL:?N8N_GOOGLE_CALLBACK_URL is required}
+      - N8N_GOOGLE_SCOPES=email profile
+      - N8N_AUTH_TYPE=google
+    volumes:
+        - /home/ubuntu/volumes/certs:/home/node/certs:ro
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)